src/index.js

```diff
@@ -33,15 +33,17 @@ if (!Promise) {
 
 const renderApp = () => {
   ReactDOM.render(
-    <I18nextProvider i18n={i18n}>
-      <Provider store={store}>
-        <PersistGate loading={null} persistor={persistor}>
-          <Router>
-            <App />
-          </Router>
-        </PersistGate>
-      </Provider>
-    </I18nextProvider>,
+    <Suspense fallback="Loading">
```

Comment:
nice - just looked up `Suspense`. It has quite an ominous name, but seems very cool

Reply:
Yeah.  I still need to research more.  Adding it prevents the page from rendering until the translations are loaded.  Otherwise a double load of the app happens